pyproject.toml

```diff
@@ -0,0 +1,14 @@
+[build-system]
+requires = ["release-gitter[builder]"]
+build-backend = "pseudo_builder"
+
+[tool.release-gitter]
+git-url = "https://github.com/Kampfkarren/selene"
+extract-files = ["selene"]
+format = "selene-{version}-{system}.zip"
+exec = "chmod +x selene"
+
+[tool.release-gitter.map-system]
+Darwin = "macos"
+Windows = "windows"
+Linux = "linux"
```

Comment:
These should no longer be needed. This was common enough that I added it as default values. That said, there is no harm in leaving them.

https://github.com/IamTheFij/release-gitter/blob/main/release_gitter.py#L51
```python
SYSTEM_SYNONYMS: list[list[str]] = [
    ["Darwin", "darwin", "MacOS", "macos", "macOS"],
    ["Windows", "windows", "win", "win32", "win64"],
    ["Linux", "linux"],
]
```

Reply:
Removed this in the latest commit ([391b18d](https://github.com/Kampfkarren/selene/pull/565/commits/391b18dead24fa6bfc6f446f814a91d4edfa6e0f)).